<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>progresswindow</class>
 <widget class="QMainWindow" name="progresswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>605</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>605</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>typeRush</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;harmonize&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 43, 140);</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>850</width>
     <height>605</height>
    </size>
   </property>
   <widget class="QGroupBox" name="settings">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>881</width>
      <height>605</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>850</width>
      <height>605</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #e2fbff;
border-radius: 10px</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <widget class="QGroupBox" name="settings_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>315</width>
       <height>362</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>315</width>
       <height>362</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(205, 235, 239);
border-radius: 10px</string>
     </property>
     <widget class="QGroupBox" name="record_parameter_groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>251</width>
        <height>291</height>
       </rect>
      </property>
      <widget class="QLabel" name="wpm_label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Baloo Chettan 2</family>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
color: rgb(255, 255, 255);
background-color:#00dacc</string>
       </property>
       <property name="text">
        <string>WPM</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="recent_stats_label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Baloo Chettan 2</family>
         <pointsize>24</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 135, 126);</string>
       </property>
       <property name="text">
        <string>Recent stats</string>
       </property>
      </widget>
      <widget class="QLabel" name="accuracy_label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Baloo Chettan 2</family>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
color: rgb(255, 255, 255);
background-color:#00dacc</string>
       </property>
       <property name="text">
        <string>Accuracy</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="nchars_label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Baloo Chettan 2</family>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
color: rgb(255, 255, 255);
background-color:#00dacc</string>
       </property>
       <property name="text">
        <string>No. of chars.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="timestamp_label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Baloo Chettan 2</family>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
color: rgb(255, 255, 255);
background-color:#00dacc</string>
       </property>
       <property name="text">
        <string>Timestamp</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="wpm_field">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Baloo Chettan 2</family>
         <pointsize>18</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:8px;
border: 5px solid #00dacc;
color: rgb(56, 157, 150)
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLabel" name="accuracy_field">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Baloo Chettan 2</family>
         <pointsize>18</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:8px;
border: 5px solid #00dacc;
color: rgb(56, 157, 150)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLabel" name="nchar_field">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Baloo Chettan 2</family>
         <pointsize>17</pointsize>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:8px;
border: 5px solid #00dacc;
color: rgb(56, 157, 150)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLabel" name="timestamp_field">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>230</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Baloo Chettan 2</family>
         <pointsize>18</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:8px;
border: 5px solid #00dacc;
color: rgb(56, 157, 150)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
      </widget>
      <zorder>accuracy_label</zorder>
      <zorder>wpm_label</zorder>
      <zorder>recent_stats_label</zorder>
      <zorder>nchars_label</zorder>
      <zorder>timestamp_label</zorder>
      <zorder>wpm_field</zorder>
      <zorder>accuracy_field</zorder>
      <zorder>nchar_field</zorder>
      <zorder>timestamp_field</zorder>
     </widget>
    </widget>
    <widget class="QFrame" name="plotframe">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>441</width>
       <height>381</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>210</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(214, 245, 249)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="homebutton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>500</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Baloo Chettan 2</family>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #00dacc;
color: rgb(255, 255, 255);
border-radius: 15px;
</string>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
